src/APIFunctions/Event.js

```diff
@@ -48,7 +48,7 @@ export async function createNewEvent(newEvent, token) {
     startTime: handleMidnightTime(newEvent.startTime),
     endTime: handleMidnightTime(newEvent.endTime),
     eventCategory: newEvent.eventCategory,
-    imageURL: newEvent.imageURL
+    imageURL: handleImageURL(newEvent.imageURL)
```

Comment:
define `handleImageURL` function above where you use it, like

```js
foo()

function foo() {
  // ...
}
```

valid:
```js
function foo() {
  // ...
}

foo()
```

Reply:
Done. Should I do the same for the handleMidnightTime function as well? I did not code this function but it is defined below the function it is used.

Reply:
First rebase your branch, it may have already been changed